simulator/dummy_perception_publisher/src/node.cpp

```diff
@@ -134,6 +166,13 @@ DummyPerceptionPublisherNode::DummyPerceptionPublisherNode()
     "input/object", 100,
     std::bind(&DummyPerceptionPublisherNode::objectCallback, this, std::placeholders::_1));
 
+  // optional ground truth publisher
+  if (publish_ground_truth_objects_) {
+    ground_truth_objects_pub_ =
+      this->create_publisher<autoware_auto_perception_msgs::msg::TrackedObjects>(
+        "output/ground_truth_objects", qos);
+  }
+
```

Comment:
```suggestion
  ground_truth_objects_pub_ =
    this->create_publisher<autoware_auto_perception_msgs::msg::TrackedObjects>(
        "~output/debug/ground_truth_objects", qos);

```

Reply:
Also done in https://github.com/autowarefoundation/autoware.universe/pull/3731/commits/a03800be2915f12c08e48b5b86b9abbdf093941b.